tests/unit-tests/templates/sample-footer-with-jinja.tpl

```diff
@@ -0,0 +1 @@
+footer content {{ variable }}
```

Comment:
(optional)

In these examples and test code, can these be updated to include the new line at the end of these files.

Reply:
I have done this, Can I suggest that you perhaps include a `.editorconfig`? (See here: https://editorconfig.org/) That would allow you to explicitly stipulate the need for new lines :-)